samples/client/petstore/lua/lua-petstore/.gitignore

```diff
@@ -0,0 +1,40 @@
+# Compiled Lua sources
```

Comment:
Not sure where you got this .gitignore file from.. it's tremendously broad. You probably don't need one at all..